homeassistant/components/cloud/__init__.py

```diff
@@ -128,7 +127,6 @@
                 vol.Optional(CONF_ACCOUNT_LINK_SERVER): str,
                 vol.Optional(CONF_ACCOUNTS_SERVER): str,
                 vol.Optional(CONF_ACME_SERVER): str,
-                vol.Optional(CONF_ALEXA_SERVER): str,
```

Comment:
Do we need a deprecation strategy for this removal?

Reply:
I don't think so. No one should be setting this value except, maybe, the developers, and for them the servicehandlers server will be picked up automatically.

Reply:
Ok, let's go ahead then.